openforcefield/tests/test_forcefield.py

```diff
@@ -251,8 +260,246 @@ def test_parameterize_ethanol_to_parmed(self):
 
         parmed_system = forcefield.create_parmed_structure(topology, positions=pdbfile.getPositions())
 
-# from_filename
-# from xml_string
+    def test_charges_from_molecule(self):
+        """Test skipping charge generation and instead getting charges from the original Molecule"""
+        # Create an ethanol molecule without using a toolkit
+        mol = Molecule()
+        mol.add_atom(6, 0, False) # C0
```

Comment:
Instead of these very long (20-line!) blocks to create a simple molecule like cyclohexane with partial charges just to test `forcefield.create_openmm_system(topology, charge_from_molecules=molecules), what about using the toolkit to do
```python
mol = OpenEyeToolkitWrapper.from_file('ethanol_with_partial_charges.mol2')
```
?

Reply:
Good question. To make these more of "unit" tests, I wanted to remove toolkit dependent behavior outside of the precise functionality being tested (in this case, parameter assignment and partial charge pass-through). Since reading any supported file (`mol2` or `sdf`) will go through a toolkit, that introduces another source of potential failure for this test. 

Once the 1.0.0 release is made and we have a stable serialized molecule format, we could use that as a source of toolkit-independent test molecules. 

To remove text bloat, I could also define these in the test setup at the top of the file, or in a different file altogether. What do you think?

Reply:
> To remove text bloat, I could also define these in the test setup at the top of the file, or in a different file altogether. What do you think?

That's a great idea. You can have a `create_test_molecule()` method that localizes all of this information into one place, and then streamline this after release.

Reply:
Separated code into `create_ethanol()` and `create_cyclohexane()` functions, and defined them at the top of `test_forcefield`